aisynphys/ui/notebook.py

```diff
@@ -233,6 +242,42 @@ def get_metric_data(metric, db, pre_classes=None, post_classes=None, pair_query_
     pairs_has_metric = pairs[~pairs[metric].isnull()]
     return pairs_has_metric, metric_name, units, scale, alpha, cmap, cmap_log, clim, cell_fmt
 
+def scatter_plots(metrics, db, pair_classes, pair_query_args, ax):
```

Comment:
Would prefer a more descriptive name that `scatter_plots`, and documentation on the function arguments.